plugins/woocommerce-admin/client/marketing/overview-multichannel/Campaigns/Campaigns.tsx

```diff
@@ -0,0 +1,170 @@
+
+	const getContent = () => {
+		if ( loading ) {
+			return (
+				<TablePlaceholder
+					caption={ tableCaption }
+					headers={ tableHeaders }
+					numberOfRows={ perPage }
+				/>
+			);
+		}
+
+		if ( ! data ) {
+			return (
+				<CardBody className="woocommerce-marketing-campaigns-card__content">
+					<Icon
+						className="woocommerce-marketing-campaigns-card__content-icon woocommerce-marketing-campaigns-card__content-icon--error"
+						icon={ cancelCircleFilled }
+						size={ 32 }
+					/>
+					<div className="woocommerce-marketing-campaigns-card__content-title">
+						{ __( 'An unexpected error occurred.', 'woocommerce' ) }
+					</div>
+					<div className="woocommerce-marketing-campaigns-card-body__content-description">
+						{ __(
+							'Please try again later. Check the logs if the problem persists. ',
+							'woocommerce'
+						) }
+					</div>
+				</CardBody>
+			);
+		}
+
+		if ( data.length === 0 ) {
+			return (
+				<CardBody className="woocommerce-marketing-campaigns-card__content">
+					<Icon
+						className="woocommerce-marketing-campaigns-card__content-icon woocommerce-marketing-campaigns-card__content-icon--empty"
+						icon={ megaphone }
+						size={ 32 }
+					/>
+					<div className="woocommerce-marketing-campaigns-card__content-title">
+						{ __(
+							'Advertise with marketing campaigns',
+							'woocommerce'
+						) }
+					</div>
+					<div className="woocommerce-marketing-campaigns-card__content-description">
+						{ __(
+							'Easily create and manage marketing campaigns without leaving WooCommerce.',
+							'woocommerce'
+						) }
+					</div>
+				</CardBody>
+			);
+		}
+
+		return (
+			<Table
+				caption={ tableCaption }
+				headers={ tableHeaders }
+				rows={ data.map( ( el ) => {
+					return [
+						{
+							display: (
+								<Flex gap={ 4 }>
+									<FlexItem className="woocommerce-marketing-campaigns-card__campaign-logo">
+										<img
+											src={ el.icon }
+											alt={ el.channelName }
+											width="16"
+											height="16"
+										/>
+									</FlexItem>
+									<FlexBlock>
+										<Flex direction="column" gap={ 1 }>
+											<FlexItem className="woocommerce-marketing-campaigns-card__campaign-title">
+												<Link href={ el.manageUrl }>
+													{ el.title }
+												</Link>
+											</FlexItem>
+											{ el.description && (
+												<FlexItem className="woocommerce-marketing-campaigns-card__campaign-description">
+													{ el.description }
+												</FlexItem>
+											) }
+										</Flex>
+									</FlexBlock>
+								</Flex>
+							),
+						},
+						{ display: el.cost },
+					];
+				} ) }
+			/>
+		);
+	};
+
+	return (
+		<Card className="woocommerce-marketing-campaigns-card">
+			<CardHeader>
+				<CardHeaderTitle>
+					{ __( 'Campaigns', 'woocommerce' ) }
+				</CardHeaderTitle>
+			</CardHeader>
+			{ getContent() }
+			{ total && total > perPage && (
```

Comment:
💅  In relation with my comment in https://github.com/woocommerce/woocommerce/pull/36735/files#r1119791419

Returning 0 instead of `undefined` feels more consistent (only one return type)  + we don't need this `total && total > perPage && ` and we can just do `total > perPage`

Reply:
Even do if we don't modify the `getTotalFromResponse` return type.... this could be reduced to 

`{ total > perPage && (`

Since `undefined` > `5` is `false`.

Reply:
> Returning 0 instead of `undefined` feels more consistent (only one return type) + we don't need this `total && total > perPage && ` and we can just do `total > perPage`

It may be convenient, but I think we may be sacrificing other things, e.g. we wouldn't be able to tell whether the API returns the value `0`, or the API did not return the value. It is better to have separate `undefined` and `0` to make things clearer.

> Even if we don't modify the `getTotalFromResponse` return type.... this could be reduced to
> `{ total > perPage && (`
> Since `undefined` > `5` is `false`.

TypeScript shows error when we do that.

![image](https://user-images.githubusercontent.com/417342/222250981-5e64ca9d-a937-48cc-b13a-5973cf714188.png)